pkgs/development/ocaml-modules/camlimages/default.nix

```diff
@@ -9,4 +9,12 @@ stdenv.mkDerivation rec {
   buildInputs = [ findlib jbuilder ocaml configurator cppo lablgtk ];
   buildPhase = "jbuilder build -p camlimages";
   inherit (jbuilder) installPhase;
+  
+  meta = with stdenv.lib; {
+    branch = "5.0";
+    homepage = https://bitbucket.org/camlspotter/camlimages;
+    description = "OCaml image processing library";
+    license = licenses.gpl2;
+    maintainers = [ maintainers.vbgl maintainers maintainers.mt-caret ];
```

Comment:
two maintainers seems fine.

Reply:
true (as you might suspect, I'm a bit tired today)